packages/babel-plugin-proposal-optional-chaining/test/fixtures/transparent-expr-wrappers/ts-as-function-call-loose/output.js

```diff
@@ -1,3 +1 @@
-var _bar, _ref;
-
-(_bar = ((_ref = (foo as A)).bar as B)) == null ? void 0 : _bar.call(_ref, foo.bar, false);
+((foo as A).bar as B) == null ? void 0 : ((foo as A).bar as B)(foo.bar, false);
```

Comment:
This is a bug: if `foo.bar` is a getter and returns an object, it will be executed twice.

Reply:
I believe this is supposed to test that the transform goes through transparent wrappers, i.e. these two two branches in the code:
https://github.com/babel/babel/blob/b2d9156cc62d37f4c522c9505a00f50b99a1eb74/packages/babel-plugin-proposal-optional-chaining/src/transform.js#L118-L122
https://github.com/babel/babel/blob/b2d9156cc62d37f4c522c9505a00f50b99a1eb74/packages/babel-plugin-proposal-optional-chaining/src/transform.js#L144-L147

In other words, stripping types from the output should give you exactly what you'd get if you stripped types from the input and then ran the transform:
```ts
// input.ts
foo.bar?.(foo.bar, false);

// output.js
foo.bar == null ? void 0 : foo.bar(foo.bar, false);
```
no temporary, no `.call`

like in this other test with no types:
https://github.com/babel/babel/blob/b2d9156cc62d37f4c522c9505a00f50b99a1eb74/packages/babel-plugin-proposal-optional-chaining/test/fixtures/assumption-pureGetters/function-call/input.js#L5
https://github.com/babel/babel/blob/b2d9156cc62d37f4c522c9505a00f50b99a1eb74/packages/babel-plugin-proposal-optional-chaining/test/fixtures/assumption-pureGetters/function-call/output.js#L5

Reply:
Oh right, `loose` implies `pureGetters`. Thanks!